src/update_engine/omaha_request_params.h

```diff
@@ -152,6 +153,7 @@ class OmahaRequestParams {
   std::string hwid_;  // Hardware Qualification ID of the client
   std::string bootid_;  // Kernel generated guid that identifies this boot
   std::string machineid_; // Unique machine ID that is set during installation 
+  std::string machinealias_; // Custom alias for the machine (optional, the empty will be ignored)
```

Comment:
"the empty" is broken grammar. Maybe "an empty value" ? or "empty values"?

Reply:
thanks, a word got lost, I wanted to say "the empty string" :)

Reply:
updated the commit